### ISSUE TITLE
zipper zips it self until no harddisk space, if set Artifact Location to ./

### ISSUE THREAD

**Issue by sabrinaluo** [XT-i/aws-lambda-jenkins-plugin#41]
It works well with below folder structure, and set Artifact Location to `./folderA`
```
repoFolder
└───folderA
    ├───file1
    └───file2
```

Error happens with below folder structure, and Artifact Location to `./`
```
repoFolder
├───file1
├───file2
└───...
```

**Comment by cast** [comment]
Reproduced, fix underway for release 0.5.3

I will revert back to using java.io.tmpdir storage instead of workspace storage for the temporary zip. I will also delete the temporary file when it is no more needed to avoid increased disk space usage.

**Comment by cast** [comment]
fixed in release 0.5.3
